### PR TITLE
Add --paper-fab-label property

### PR DESCRIPTION
**Pull request by sorin-davidoi**
Fixes: #67 

Add `--paper-fab-label` custom property.

### CODE REVIEW

File: paper-fab.html [hunk 1]
```diff
@@ -114,6 +115,8 @@
         overflow: hidden;
         text-overflow: ellipsis;
         text-align: center;
+
+        @apply(--paper-fab-label)
```
Comment (keanulee) [inline paper-fab.html]: Convention: missing semicolon

**Comment by sorin-davidoi** [comment]
Fixed.

File: paper-fab.html (hunk 1 above)
Comment (sorin-davidoi) [inline paper-fab.html]: Fixed.

**Comment by keanulee** [comment]
LGTM. Thanks @sorin-davidoi!

**Pull request merged**